examples/step-74/step-74.cc

```diff
@@ -202,7 +202,7 @@ namespace Step74
                          const double       cell_extend_left,
                          const double       cell_extend_right)
   {
-    const double degree = std::max(1., static_cast<double>(fe_degree));
+    const double degree = std::max<double>(1, fe_degree);
```

Comment:
I am not a big fan of this change: It makes it less obvious that a conversion int->double is happening and spells out "double" twice.

Reply:
I changed that further in #11467. Let's not worry about it here.